File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.java
class Solution {
    public int solution(int a, int b) {
        String astring = String.valueOf(a);
        String bstring = String.valueOf(b);
        int add = Integer.valueOf(astring+bstring);
        if(add > (2*a*b)){
            return add;
        }
        else{
            return 2*a*b;
        }
    }
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.cs
using System;

public class Solution {
    public int solution(int a, int b) {
        string astring = a.ToString();
        string bstring = b.ToString();
        int add = int.Parse(astring + bstring);
        
        if (add > (2 * a * b)) {
            return add;
        } else {
            return 2 * a * b;
        }
    }
}

